src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -258,7 +267,7 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
                     chartControls={this.chartControls}
                     changeChartType={this.changeChartType}
                 />
-                <StudyViewComponentLoader promise={this.props.promise}>
+                <StudyViewComponentLoader promise={this.props.promise} isInitialLoaded={!_.isUndefined(this.chartData)}>
```

Comment:
You can now just use === undefined.     The problem with testing against undefined is old and has been solved in modern browsers.